addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -534,6 +534,15 @@ var SnippetEditor = Widget.extend({
                 } else {
                     $optionsSection.append(option.$el);
                 }
+                if (option.$el !== undefined) {
```

Comment:
How would this be possible? :) Anyways, this is code should not be placed here but inside the widget declaration themself. Like in the `start` method of `UserValueWidget`. Or maybe on the whole snippet menu to only bind one event to handle all at once... I let you see.